playbook/spec/pb_kits/playbook/kits/text_input_spec.rb

```diff
@@ -19,12 +19,12 @@
 
   describe "#classname" do
     it "returns namespaced class name", :aggregate_failures do
-      expect(subject.new({}).classname).to eq "pb_text_input_kit"
-      expect(subject.new(classname: "additional_class").classname).to eq "pb_text_input_kit additional_class"
-      expect(subject.new({ dark: true }).classname).to eq "pb_text_input_kit dark"
-      expect(subject.new({ inline: true }).classname).to eq "pb_text_input_kit inline"
-      expect(subject.new({ error: "Please enter a valid email" }).classname).to eq "pb_text_input_kit error"
-      expect(subject.new({ dark: true, error: "Please enter a valid email" }).classname).to eq "pb_text_input_kit dark error"
+      expect(subject.new({}).classname).to eq "pb_text_input_kit mb_sm"
```

Comment:
Every single spot where people are targeting our pb classname could break.
There are at least 24 here that would need to be thoroughly tested by ninjas: https://github.com/search?q=repo%3Apowerhome%2Fnitro-web+pb_text_input_kit&type=code